checks/utils/windowsversions.py

```diff
@@ -17,20 +17,21 @@
     """, re.VERBOSE)
 
 
-# Ref: https://docs.microsoft.com/en-us/lifecycle/products/windows-11-home-and-pro-version-21h2
+# Ref: https://docs.microsoft.com/lifecycle/products/windows-11-home-and-pro
+#      https://aka.ms/Windows11UpdateHistory
+#      https://docs.microsoft.com/windows/release-health/windows11-release-information
 win11versions = {
     22000: {
         "release": 2009,
         "name": "Windows 11 21H2",
         "date": datetime.date(2021, 10, 4),
         "EoS": datetime.date(2023, 10, 10),
     },
-    # Note: 22622 is 22H2 but with additional features coming 2023
```

Comment:
Why remove this?

Reply:
The current build number for Windows 11 22H2 beta channel is 22631. And there are also 22623 and 22624 between 22622 and 22631. Unless the beta channel is also added, this comment can be deleted.
https://blogs.windows.com/windows-insider/2023/05/25/announcing-windows-11-insider-preview-build-22621-1825-and-22631-1825/

Reply:
The comment is just informative, as far as I know. 22622 (and up, for now) will also identify as "Windows 11" and might even show as "22H2" in some tools. The comment itself is not meant for us to add entries for Insider revisions/builds.